packages/app/obojobo-repository/shared/actions/dashboard-actions.test.js

```diff
@@ -10,11 +10,16 @@ dayjs.extend(advancedFormat)
 describe('Dashboard Actions', () => {
 	let DashboardActions
 	let mockFileReader
+	let undefined
+
+	const JSON_MIME_TYPE = 'application/json'
+	const XML_MIME_TYPE = 'application/xml'
```

Comment:
Since there are now two valid XML mime types, make sure you test that both of them work.